mods/blocks/fire.lua

```diff
@@ -0,0 +1,247 @@
+local S = default.get_translator
+
+blocks.fire_types = {
+    {"amber", "Amber", "blocks:amber_shard"},
+    {"amethyst", "Amethyst", "blocks:amethyst_crystal"},
+    {"azure", "Azureshroom", "group:azure"},
+    {"diamond", "Diamond", "blocks:diamond"},
+    {"fire", "Fireshroom", "group:fire_biome"},
+    {"garnet", "Garnet", "blocks:garnet_crystal"},
+    {"malachite", "Malachite", "blocks:malachite"},
+    {"mese", "Mese", "blocks:mese_crystal"},
+    {"moonstone", "Moonstone", "blocks:moonstone_crystal"},
+    {"lapis", "Lapis Lazuli", "blocks:lapis_lazuli"}
+}
+
+-- Fires
+
+minetest.register_node("blocks:fire", {
+    description = S("Fake Fire"),
+    tiles = {{
+        name = "blocks_fire_animated.png",
+        animation = {type = "vertical_frames", aspect_w = 16, aspect_h = 16, length = 2.0}
+    }},
+    inventory_image = "blocks_fire.png",
+    drawtype = "firelike",
+    paramtype = "light",
+    sunlight_propagates = true,
+    walkable = false,
+    floodable = true,
+    light_source = default.LIGHT_MAX,
+    groups = {oddly_breakable_by_hand = 1},
+    --sounds = default.node_sound_leaves_defaults()
```

Comment:
Accidental comment or just remove it, seems like an unnecessary comment otherwise.

Reply:
Do we have any sound collection that would fit these?

Reply:
We don't, if you want to import the fire sounds for MTG fire mod.
On the case of sound as well, I'm not sure if you want to add like active fire sounds to it as well.
I'm neutral for both, and am fine with no sounds.

Reply:
I see. I thought about adding sounds and I think they are out of the scope of this branch. I'll remove the comments